.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: cargo
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
Probably should've mentioned this option...

With `Cargo.lock` checked in, PRs to update it like #212 can create a lot of churn, especially for crates that update frequently.

If that gets to be a problem, we can turn this down to avoid bumping several point releases in a row.